testing/test-vectors/prng-tests.lisp

```diff
@@ -2,5 +2,8 @@
 (in-package :crypto-tests)
 
 (rtest:deftest :prng-fortuna (run-test-vector-file :prng *prng-tests*) t)
+(rtest:deftest :prng-fortuna-urandom (let ((prng (crypto:make-prng :fortuna :seed urandom)))
```

Comment:
Some tests fail in CCL 1.10 (the version used by the travis-ci script), but everything works fine in CCL 1.11 (the version on my system).